src/data/markdown/translated-guides/en/02 Using k6/06 Test life cycle.md

```diff
@@ -83,29 +112,27 @@ export default function () {
 
 ## The default function life-cycle
 
-A VU will execute the default function from start to end in sequence. Nothing out of the ordinary
-so far, but here's the important part; once the VU reaches the end of the default function it will
-loop back to the start and execute the code all over.
+A VU executes the default function from start to end in sequence.
+Once the VU reaches the end of the default function, it loops back to the start and executes the code all over.
 
-As part of this "restart" process, the VU is reset. Cookies are cleared and TCP connections
-might be torn down, depending on your test configuration options.
+As part of this "restart" process, k6 resets the VU.
+Cookies are cleared and TCP connections
+might be torn down  (depending on your test configuration options).
 
-> Make sure to use `sleep()` statements to pace your VUs properly. An appropriate amount of
-> sleep/think time at the end of the default function is often needed to properly simulate a
-> user reading content on a page. If you don't have a `sleep()` statement at the end of
-> the default function your VU might be more "aggressive" than you've planned.
+> Make sure to use `sleep()` statements to pace your VUs properly, simulating a user reading content on your page.
+> If you don't have a `sleep()` statement at the end of the default function, your VU might be more "aggressive" than you've planned.
 >
 > VU without any `sleep()` is akin to a user who constantly presses F5 to refresh the page.
 
 ## Setup and teardown stages
 
-Beyond the required init and VU stages, which is code run for each VU, k6 also supports test-wide
-setup and teardown stages, like many other testing frameworks and tools. The `setup` and
-`teardown` functions, like the `default` function, needs to be exported functions. But unlike
-the `default` function `setup` and `teardown` are only called once for a test. `setup` is called
-at the beginning of the test, after the init stage but before the VU stage (`default` function),
-and `teardown` is called at the end of a test, after the VU stage (`default` function). Therefore,
-VU number is 0 while executing the `setup` and `teardown` functions.
+Like `default`, `setup` and `teardown` functions must be exported functions.
+But unlike the `default` function, k6 calls `setup` and `teardown` only once per test. 
+
+* `setup` is called at the beginning of the test, after the init stage but before the VU stage.
+* `teardown` is called at the end of a test, after the VU stage (`default` function).
+
+The VU number is 0 while the `setup` and `teardown` functions execute.
```

Comment:
I am unsure this belongs here. It is a very specific detail and looks out of context here.

I would consider it moving it together with:

- Note that the end-of-test summary includes requests made in the setup and teardown stages.
- Skip setup and teardown execution
- The VU number is 0 while the `setup` and `teardown` functions execute